sdk/src/wallet/account/operations/helpers/time.rs

```diff
@@ -14,26 +14,25 @@ pub(crate) fn can_output_be_unlocked_now(
     // We use the addresses with unspent outputs, because other addresses of the account without unspent outputs can't
     // be related to this output
     account_addresses: &[AddressWithUnspentOutputs],
-    alias_and_nft_addresses: &[Address],
+    account_and_nft_addresses: &[Address],
```

Comment:
Might also change, but looks confusing to have account_addresses and then account_and_nft_addresses

Reply:
Added a TODO